code/workspaces/client-api/src/dataaccess/login/rstudioTokenService.js

```diff
@@ -14,7 +15,7 @@ const rstudioLogin = notebook => (credentials) => {
 };
 
 const login = (notebook, credentials) => (response) => {
-  logger.debug(`Logging in to RStudio instance at: ${notebook.internalUrl}`);
+  logger.debug(`Logging in to RStudio instance at: ${getRStudioLoginUrl(notebook)}`);
```

Comment:
Doesn't really matter but the equivalent log message would be `getCorrectAccessUrl(notebook)` I guess.

Reply:
That's a good point, though it might be good to have the actual URL that the function is going to be hitting logged?